didman/didman.go

```diff
@@ -524,6 +572,33 @@ func (d *didman) addService(ctx context.Context, id did.DID, serviceType string,
 	return service, nil
 }
 
+func (d *didman) updateService(ctx context.Context, id did.DID, serviceType string, serviceEndpoint interface{}) (*did.Service, error) {
+	doc, _, err := d.docResolver.Resolve(id, nil)
+	if err != nil {
+		return nil, err
+	}
+	service := &did.Service{
+		Type:            serviceType,
+		ServiceEndpoint: serviceEndpoint,
+	}
+	service.ID = generateIDForService(id, *service)
+
+	serviceFound := false
```

Comment:
Can you add a comment what this serviceFound does? Perhaps add a function description as well and under which conditions it might error.

Reply:
changed variable name to `serviceToBeUpdatedFound`, function is quite self-explanatory i.m.o?